setup.py

```diff
@@ -23,18 +23,18 @@
         'treelib<=1.5.5',
         'pymining',
         'sklearn',
-        'git+https://github.com/maciejkula/spotlight.git',
+        'spotlight @ git+git://github.com/maciejkula/spotlight.git@v0.1.6#egg=spotlight',
```

Comment:
I think you don't need to install spotlight here, do you ?It's only developed by a recommender we do in house, and if we port that to this repo, we would do as part of a separate pr.